spec/content_tagger/removing_taxon_spec.rb

```diff
@@ -0,0 +1,37 @@
+feature "Removing content from Content Tagger", new: true, collections: true, content_tagger: true do
+  include ContentTaggerHelpers
+
+  let(:redirection_destination_title) { "Removed taxon destination " + SecureRandom.uuid }
+  let(:redirection_destination_slug) { "redirection-taxon-" + SecureRandom.uuid }
+  let(:removed_slug) { "removed-taxon-" + SecureRandom.uuid }
+
+  scenario "Unpublishing a taxon" do
+    given_there_are_two_published_taxons
+    and_i_remove_one_by_redirecting_to_the_other
+    then_visiting_the_removed_taxon_redirects_to_the_other_taxon
+  end
+
+  def given_there_are_two_published_taxons
+    create_draft_taxon(slug: redirection_destination_slug, title: redirection_destination_title)
+    publish_taxon
+    @redirection_destination_url = find_link("/" + redirection_destination_slug)[:href]
+    reload_url_until_status_code(@redirection_destination_url, 200)
+
+    create_draft_taxon(slug: removed_slug, title: "Removed taxon " + SecureRandom.uuid)
+    publish_taxon
+    @redirected_taxon_url = find_link("/" + removed_slug)[:href]
+    reload_url_until_status_code(@redirected_taxon_url, 200)
```

Comment:
I wonder if we want to put this into a method since we have two very similar blocks? not a big deal though.